src/main/kotlin/io/github/mojira/arisa/modules/LanguageModule.kt

```diff
@@ -0,0 +1,58 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import arrow.core.left
+import arrow.core.right
+import com.github.pemistahl.lingua.api.Language
+import com.github.pemistahl.lingua.api.LanguageDetectorBuilder
+
+class LanguageModule(
+    private val allowedLanguages: List<Language> = listOf(
+        Language.ENGLISH,
+        Language.UNKNOWN
+    )
+) : Module<LanguageModule.Request> {
+
+    companion object {
+        private val detector = LanguageDetectorBuilder.fromLanguages(
```

Comment:
I would move this out of the companion object and just have it in the class. Companion objects have some problems(they generate way to much code) and this doesnt really need to be static

Modules are initialized only once anyway